civihr_default_theme/scss/components/_header.scss

```diff
@@ -178,35 +184,47 @@
   }
 
   .menu-link-hr-admin {
-    order: 9999;
+    line-height: $chr-hr-admin-margin;
+    order: 9999; // to ensure it appears the last in the list
+    padding: 12px 0 0 15px;
```

Comment:
Same, needs a variable I think. Also, where does this value come from? If **not** from a designer, can we contact them and ask what the padding should be?